app/models/audit/AuditInfo.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.audit
+
+import play.api.libs.json.{Json, OFormat}
+
+case class AuditInfo(pid: String,
```

Comment:
There is more information here than defined in Confluence at https://confluence.tools.tax.service.gov.uk/display/OCEG/External+Guidance+-+TxM+Assessment
What is the official requirement?

Reply:
as we talked about earlier, content was whatever was most useful to us as TXM have no requirements for us to meet.  Added a few more ocelot fields as discussed at one of the standups last week and will updatet he confluence page to match once the chnage has been pushed up